.yamato/config/__shared.metafile

```diff
@@ -58,9 +75,10 @@ platforms:
     apis:
       - name: Metal
         exclude_test_platforms:
-          - editmode
```

Comment:
why do we allow editmode now or it's not intentional?

Reply:
because the performance build tests are editmode tests

Reply:
true, but we call these performance build tests with some other name, right? i think the matching is done by testplatform name, i.e. 'if testplatform[name] == editmode then skip", so removing this line means all normal editmode jobs also get created (before we threw exception in the _cmd functions, which made it impossible to use editmode for performance, but we removed this i think), which is why I think we should add this back (the performance jobs should not be affected as long as they are not named editmode)

Reply:
i see your point - and to be fair, im not even enabling the build jobs in this pr, so i can add this bak in anyway and we can figure it out after